eng/verify-version.ps1

```diff
@@ -0,0 +1,62 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory)]
+    [string] $VSTestExe
+)
+
+$Script:RootDir = (Get-Item (Split-Path $MyInvocation.MyCommand.Path)).Parent.FullName
+
+function Extract-VSTestVersion {
+    param (
+        [string]$VSTestExe
+    )
+
+    $outputFile = Join-Path $RootDir "output.txt"
+
+    Invoke-Expression $VSTestExe > $outputFile 2>&1
```

Comment:
This is not what will fail the check in DTE, they don't care about our output. The informational version on the file will.

Reply:
This wasn't meant as a check for DTE, but we can extend it to cover that scenario too. It was originally meant to make sure we don't have unaligned packages (branch vs. version), we had a couple of instances where the suffix wasn't properly changed on the release branch and we released a `preview` version from a `rel/*` branch. The script was designed to prevent such scenarios from re-occurring.

Reply:
vstest.console output takes it from the assembly informational version, which you can check directly on the file metadata. Which is also what DTE checks.  

Please use this instead of parsing output. It will also avoid reporting RTM version, because that what release versions do by default in output (you are not specifying `--diag`).
```powershell
(Get-Item .\vstest.console.dll).VersionInfo.ProductVersion
```

Reply:
Fixed, thanks. Now we're checking only `ProductVersion` as per your suggestion.